de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/ISarosSession.java

```diff
@@ -387,26 +387,17 @@ public void addSharedResources(
   public void changeColor(int colorID);
 
   /**
-   * FOR INTERNAL USE ONLY !
-   *
-   * <p>Starts queuing of incoming {@linkplain IResourceActivity project-related activities}, since
-   * they cannot be applied before their corresponding project is received and extracted.
+   * Register a queue handler for an incoming project negotiation. Should be called only once or
+   * always with the same <code>activityQueue</code>, otherwise the caller is responsible for making
+   * sure that the handler can be safely changed (empty old queue).
    *
-   * <p>That queuing relies on an existing project-to-projectID mapping (see {@link
-   * #addProjectMapping(String, IProject)}), otherwise incoming activities cannot be queued and will
-   * be lost.
-   *
-   * @param project the project for which project-related activities should be queued
-   * @see #disableQueuing
+   * @client
```

Comment:
This is not a valid JavaDoc annotation and we do not have any JavaDoc preprocessor.

If you find "pearls" like "@host, @client, @non-swt ... etc" just deleted them and add an accurate description to the existing JavaDoc.